environment/README.md

```diff
@@ -6,7 +6,7 @@ steps to create a conda environment and download the necessary data.
 Keep in mind that the software only runs on Linux or macOS, **Windows is not
 supported** now.
```

Comment:
```suggestion
supported** yet.
```
You could also just leave this last word off, or I would potentially just phrase it as "Windows is not currently supported"